scripts/prepare_sector_network.py

```diff
@@ -1961,12 +1995,14 @@ def add_dac(n, costs):
     ].index  # TODO if you take nodes from the index of buses of n it's more than pop_layout
     # clustering of regions must be double checked.. refer to regions onshore
 
+    n.loads.loc[nodes, "carrier"] = "AC"
+
     Nyears = n.snapshot_weightings.generators.sum() / 8760
 
     # TODO fetch investment year from config
 
     investment_year = int(snakemake.wildcards.planning_horizons[-4:])
-
+    demand_sc = snakemake.wildcards.demand

```

Comment:
what does "_sc" stand for? Not quite clear, maybe add short explaination as inline comment

Reply:
demand_sc stands for demand scenario. I added an inline comment already

Reply:
Perfect, thanks.